src/popup.html

```diff
@@ -222,6 +230,51 @@ <h2 class="fx-relay-panel-header-title i18n-content" data-i18n-message-id="popup
         </settings-panel>
 
+         <!-- Panel: CSAT Survey -->
+         <survey-panel data-panel-id="survey" class="is-hidden fx-relay-panel" id="survey-panel">
+          <header class="fx-relay-panel-header">
+            <!-- Back Button -->
+            <button data-nav-level="root" data-back-target="masks" class="fx-relay-panel-header-btn-back">
+              <img class="i18n-alt-tag" data-i18n-message-id="returnToPreviousPanel" src="/icons/nebula-back-arrow.svg" alt="">
+            </button>
+            
+            <!-- Panel Title -->
+            <h2 class="fx-relay-panel-header-title i18n-content" data-i18n-message-id="popupSurveyLink" />
+
+          </header>
+          
+          <div class="fx-relay-panel-content">
+             <!-- Satisfaction Selection Form -->
+            <div class="csat-survey-content">
+              <div class="csat-survey-section">
+                <p class="csat-survey-text i18n-content" data-i18n-attribute="title" data-i18n-message-id="popupCsatSurveyText" /> 
+              </div>
+
+              <div class="csat-survey-satisfaction-section">
+                <!-- Select issue checkboxes -->
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="0" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionVeryDissatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="1" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionDissatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="2" data-event-action="click"  data-i18n-message-id="popupCsatSurveySatisfactionNeutral"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="3" data-event-action="click"  data-i18n-message-id="popupCsatSurveySatisfactionSatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="4" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionVerySatisfied"></button> 
+
+                <!-- User made a choice about their satisfaction -->
+                <div class="fx-relay-survey-success is-hidden"> 
+                  <h1 class="i18n-content fx-relay-survey-success-title" data-i18n-message-id="popupCsatSurveyChoiceMadeTitle"></h1>
+                  <p class="i18n-content fx-relay-survey-success-body" data-i18n-message-id="popupCsatSurveyChoiceMadeBody"></p> 
+                </div>
+              </div> 
+            </div> 
+          </div>
+          
+          <div class="fx-relay-survey-actions">
+            <!-- using anchor tags for navigation -->
+            <a href="" class="i18n-content fx-relay-survey-action fx-relay-survey-dismiss" data-event-action="click" data-event-label="panel-survey-dismiss" data-i18n-message-id="popupCsatSurveyDismiss"></a>
```

Comment:
As this is an internal link, should this be a `<button>` element instead of an `<a>`?

Reply:
It's used for navigation, I would say anchor tag for that. But I'll yield to your input if you think otherwise.

Reply:
This will need some change (`aria`, `tabindex`, etc.) to become keyboard navigable. It still is not.

Reply:
IIRC `<a>` tags without an `href` are not reachable via keyboard, so the fix would be to either give it an `href`, or turn it into a `<button>`, I think.

Reply:
converted to button